### PR TITLE
Merge Tensorboard-intgration-bp with tensorboard-integration-ku

### PR DESCRIPTION
**Pull request by bprimal22**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,4 @@ bazel-*
 
 data_generation/data
 data_generation/assets
+runs
```
Comment (kadhirumasankar) [inline .gitignore]: We don't need this line. You can delete it

File: README.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ architecture. Check out `torchvision` for all the possibilities. An example
 training command is:
 ```
 PYTHONPATH=$(pwd) train/train_clf.py \
-    --model_config configs/resnet18.yaml
+    --config configs/vovnet.yaml
```
Comment (kadhirumasankar) [inline README.md:44]: @alexwitt2399 keep this change?

**Review by kadhirumasankar — CHANGES_REQUESTED** [review]
Just fix .gitignore and we'll be good to merge

File: README.md (hunk 2 above)
Comment (alexwitt23) [reply README.md:44]: yeah those changes are right

**Review by kadhirumasankar — APPROVED** [review]
Looks good to me 👍

**Pull request merged**